### ISSUE TITLE
[TF2] Ultra Long MvM queue time towards Aussie players

### ISSUE THREAD

**Issue by jackyc92789** [ValveSoftware/Source-1-Games#6816]
When there's an Aussie player in a party or solo queueing, the match-making system tends to get all Aussie players in a server. However, Here's the issue:
1. For solo queueing: There are not many Aussie players all the time. This results in an extra long queue time for whoever wants to solo queue. The system takes way too long to realize that getting 6 Aussie players is impossible (est. 10-30mins). Also, it is basically impossible for AU players to get servers that have other ping ranges. Ever servers with 100-150 ping like Asia servers or NA servers that are largely populated, I find it hard to get a server with the new match-making system.
2. For party queueing: The system recognizes a ping difference among the parties. It starts to search for a good server. For example, I was partying with some LA players and I am the only AU player. The system is incapable of figuring out what server should the players put in, resulting in long queue times.
In conclusion, Please fix the matchmaking system so that Aussie players can wait less time to find a server. I do believe If players set their match-making ping range to under 200, there should be able to find LA/Korea/HK/Tokyo more frequently than suck in the Sydney server forever.
Aussie players deserve better queue times, different server variety, and the opportunity to meet players around the globe.
